src/common/mathjax.js

```diff
@@ -53,7 +53,7 @@ goog.require('sre.System');
 
   (sre.System.getInstance()).setupEngine(
       {mode: sre.Engine.Mode.HTTP,
-       json: MathJax.Ajax.config.path['SRE'],
+       json: MathJax.Ajax.config.path['SRE'] + '/mathmaps/',
```

Comment:
Ah, thanks for finding that. I always wanted to have a utility function to take care of concatenating path names. Maybe it's time now.